fe/fe-core/src/main/java/org/apache/doris/catalog/ResourceMgr.java

```diff
@@ -52,7 +55,7 @@
     private static final Logger LOG = LogManager.getLogger(ResourceMgr.class);
 
     public static final ImmutableList<String> RESOURCE_PROC_NODE_TITLE_NAMES = new ImmutableList.Builder<String>()
-            .add("Name").add("ResourceType").add("Key").add("Value")
```

Comment:
Better don't use 'key' annd 'value' because they are keyword in doris.
For example , `show resources where name like "test" and resourcetype = "spark" order by KEY;` will report an error

Reply:
Hi,xy720. I keep the older name of the Resource. The `key` and `value` is the keyword in doris. The only way to use it is to add operator ` to prevent the error of doris. 
Maybe we should change the name.